Verification/FLiBe_Prop_Verification.py

```diff
@@ -0,0 +1,164 @@
+	#Temperature is in Kelvin
+
+import os
+import sys
+sys.path.insert(0,'..') #This adds the ability to call flibe props from the main folder
+sys.path.insert(0,'./Flibe') #Looking for data in a subfolder
+import numpy as np 
+import matplotlib.pyplot as plt  
+import FLiBe_Prop
+import Flibe_Cp_Verification_Data
+import pandas as pd
+from pathlib import Path
+
+steps = 700
+
+T = np.linspace(550,1250,steps)
+T2 = np.linspace(700,1200,steps)
+
+Rho = np.zeros(steps)
+Nu = np.zeros(steps)
+Cp = np.zeros(steps)
+K = np.zeros(steps)
+
+for i in range(0,steps):
+	Rho[i] = FLiBe_Prop.rho(T[i])
+	Nu[i] = FLiBe_Prop.nu(T2[i])*1000
+	Cp[i] = FLiBe_Prop.Cp(T2[i])
+	K[i] = FLiBe_Prop.k(T2[i])
+
+path = os.getcwd()
+data1 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Blanke_1956.csv'))
+df1 = pd.DataFrame(data1, columns=['Temp','Density'])
+data2 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Cantor_1968.csv'))
+df2 = pd.DataFrame(data2, columns=['Temp','Density'])
+data3 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Cantor_1973.csv'))
+df3 = pd.DataFrame(data3, columns=['Temp','Density'])
+data4 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Gierszewski_1980.csv'))
+df4 = pd.DataFrame(data4, columns=['Temp','Density'])
+data5 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Janz_1974.csv'))
+df5 = pd.DataFrame(data5, columns=['Temp','Density'])
+data6 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Janz_1974_1988.csv'))
+df6 = pd.DataFrame(data6, columns=['Temp','Density'])
+data7 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Recommended.csv'))
+df7 = pd.DataFrame(data7, columns=['Temp','Density'])
+data8 = pd.read_csv(Path(path + '/Flibe/Density/Romatoski_Flibe_Density_Zaghoul_2003.csv'))
+df8 = pd.DataFrame(data8, columns=['Temp','Density'])
+
+fig1 = plt.figure(1, figsize=(10,8))
+ax = fig1.add_subplot()
+df1.plot(x='Temp', y='Density', label='Blanke 1956', ax=ax, kind='scatter', marker='D', color='blue', figsize=(10,8))
+df2.plot(x='Temp', y='Density', label='Cantor 1968', ax=ax, linestyle='--', color='purple')
+df3.plot(x='Temp', y='Density', label='Cantor 1973', ax=ax, marker='^', kind='scatter', color='darkgreen')
+df4.plot(x='Temp', y='Density', label='Gierszewski 1980', ax=ax, linestyle=':', color='fuchsia')
+df5.plot(x='Temp', y='Density', label='Janz 1974', ax=ax, kind='scatter', marker='s',color='firebrick')
+df6.plot(x='Temp', y='Density', label='Janz 1974/1988', ax=ax, linestyle='--', color='darkorange')
+df7.plot(x='Temp', y='Density', label='Recommended', ax=ax, linestyle='-.', color='cornflowerblue')
+df8.plot(x='Temp', y='Density', label='Zaghoul 2003', ax=ax, linestyle='--', color='lime')
+plt.plot(T,Rho,'r--', label='Density Used', linewidth=3, dashes=(10,20))
+plt.xlabel('Temperature - K')
+plt.ylabel('Density - kg/m^3')
+plt.legend(loc='upper right')
+plt.grid()
+plt.title('Flibe Density')
+plt.show()
```

Comment:
Remove this plt.show(), only put one at the end so all of the plots show up at once.